### ISSUE TITLE
UnnecessaryModifier removes non-redundant abstract, final modifiers from nested classes

### ISSUE THREAD

**Issue by mches** [solven-eu/cleanthat#842]
`UnnecessaryModifier` removes non-redundant `abstract`, `final` modifiers from nested classes

e.g.

```java
public interface Parent {
  public static abstract class AbstractClass {}

  public static final class FinalClass extends AbstractClass {}
}
```
↓↓↓
```java
public interface Parent {
  class AbstractClass {}

  class FinalClass extends AbstractClass {}
}
```

and

```java
public @interface Parent {
  public static abstract class AbstractClass {}

  public static final class FinalClass extends AbstractClass {}
}
```
↓↓↓
```java
public @interface Parent {
  class AbstractClass {}

  class FinalClass extends AbstractClass {}
}
```

**Comment by blacelle** [comment]
Thanks for the report and the fix. This has been released in Cleanthat 1.22 ; it shall be available in m2central within minutes/hours.

**Comment by mches** [comment]
Thanks. I've already put it to use
